src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,19 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String brand;
+    private String model;
+    private int horsePower;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String brand, String model, int horsePower) {
+        this.brand = brand;
+        this.model = model;
+        this.horsePower = horsePower;
+    }
```

Comment:
When you add implementation and push Machine, you should also take params that reflect machine's fields and call it's constructor using `super(...)`

Reply:
Same for all subclasses of Machine